src/components/button/button.css

```diff
@@ -0,0 +1,153 @@
+.cx-btn__primary,
+.cx-btn__secondary,
+.cx-btn__tertiary,
+.cx-btn__danger {
+  --highlight-color: var(--cx-color-background-accent-5);
+  --highlight-background: var(--cx-color-background-accent-1);
+  position: relative;
+  display: inline-flex;
+  gap: var(--cx-spacing-1);
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  font-size: 0.875rem;
+  font-weight: 500;
+  line-height: 1rem;
+  padding: var(--cx-spacing-4) var(--cx-spacing-6);
+  min-width: 6rem;
+  border-radius: var(--cx-radius-pill);
+  transition: scale 20ms ease, background-color 300ms ease, color 300ms ease, box-shadow 300ms ease;
+  grid-template-columns: repeat(auto-fill);
+
+  &:active {
+    scale: 0.99;
+  }
+
+  &:hover,
+  &:focus-visible {
+    box-shadow: none;
+    background-color: transparent;
+  }
+
+  &::before {
+    content: "";
+    position: absolute;
+    inset: 0;
+    background-image: radial-gradient(
+      ellipse at center,
+      var(--highlight-color),
+      var(--highlight-background) 45%,
+      var(--highlight-background)
+    );
+
+    border-radius: inherit;
+    background-size: 500% 500%;
+    background-position: 70% 30%;
+    animation: cx-btn-highlight-rotate 20s linear infinite;
+    z-index: -1;
+  }
+}
+
+:is(.cx-btn__primary, .cx-btn__secondary, .cx-btn__tertiary):is(:hover, :focus-visible) {
+  color: var(--cx-color-text-inverted);
```

Comment:
Skal primary være med her, siden den allerede har color-inverted som default?

Reply:
Primary trenger strengt tatt ikke være med der nei. Det var bare litt for å gjøre det "enkelt" at jeg nevner alle knappene som ikke er "danger". Men kan fjerne primary 🙂